src/core_atmosphere/physics/mpas_atmphys_lsm_noahmpinit.F

```diff
@@ -234,7 +236,7 @@ subroutine noahmp_init(configs,mesh,diag_physics,diag_physics_noahmp,output_noah
  integer,dimension(:),pointer:: isnowxy
  integer,dimension(:),pointer:: irnumsi,irnummi,irnumfi
  
- real(kind=RKIND),pointer:: dt
+ real(kind=RKIND) :: dt
```

Comment:
Just one more minor request: could we remove the space before the `::` to match the style of other declarations in this file?

Reply:
Done